### DISCUSSION TITLE
How to config additional paths for Babel to compile

### DISCUSSION THREAD

**Discussion by yankustefan** [vercel/vercel#4500] [Help] [answered]
Hi,

I have a custom component library in the node_modules folder, which I would like to compile together with the nextjs app (at least during development)

I am using [Yalc](https://github.com/whitecolor/yalc), to work with yarn/npm packages locally.
Before, I was using create react app, and was able to accomplish that with [customize-cra](https://github.com/arackaf/customize-cra):

```
module.exports = function (config, env) {
  return Object.assign(config, override(
    disableEsLint(),
    addDecoratorsLegacy(),
    /*Make sure Babel compiles the stuff in these foldersr*/
    babelInclude([
      path.resolve('src'), // don't forget this
      path.resolve('.yalc/my-module/src'),
      path.resolve('node_modules/my-module/src')
    ])
    )(config, env)
  )
}
```

Now I would need to do that in next.config.js ?
Do I need to modify next-babel-loader ?

I would appreciate any input on how to accomplish that.

**Comment by Timer** [comment, marked as answer]
Hello! Questions related to Next.js belong in the Next.js Discussions:
https://github.com/vercel/next.js/discussions

Please re-post this there.

**Reply by yankustefan** [reply]
Thank you for pointing me to the right discussion.
I did that, [here](https://github.com/vercel/next.js/discussions/13532).
